packages/ee/billing/lib/createSubscription.ts

```diff
@@ -20,153 +20,66 @@ export const getFirstOfNextMonthTimestamp = (): number => {
 export const createSubscription = async (
   teamId: string,
   environmentId: string,
-  priceLookupKeys: StripePriceLookupKeys[]
+  priceLookupKey: StripePriceLookupKeys
 ) => {
   try {
     const team = await getTeam(teamId);
     if (!team) throw new Error("Team not found.");
     let isNewTeam =
       !team.billing.stripeCustomerId || !(await stripe.customers.retrieve(team.billing.stripeCustomerId));
 
-    let lineItems: { price: string; quantity?: number }[] = [];
-
-    const prices = (
+    const priceObject = (
       await stripe.prices.list({
-        lookup_keys: priceLookupKeys,
+        lookup_keys: [priceLookupKey],
+        expand: ["data.product"],
       })
-    ).data;
-    if (!prices) throw new Error("Price not found.");
+    ).data[0];
 
-    prices.forEach((price) => {
-      lineItems.push({
-        price: price.id,
-        ...(price.billing_scheme === "per_unit" && { quantity: 1 }),
-      });
-    });
+    if (!priceObject) throw new Error("Price not found");
+    const responses = (priceObject.product as Stripe.Product).metadata.responses as unknown as number;
```

Comment:
same as mentioned above, pls don't type cast it and actually convert it to an integer using `parseInt`